content/posts/2021/release-0.8.0.md

```diff
@@ -0,0 +1,362 @@
+finished time for each individual job in the workflow:
+
+```console
+$ reana-client logs -w my-analysis.42
+...
+==> Step: gendata
+==> Workflow ID: 4455a6b2-3d94-4694-ae99-e493327cd53f
+==> Compute backend: Kubernetes
+==> Job ID: reana-run-job-6f57f1c8-8edf-4840-9423-b638e664bf57
+==> Docker image: reanahub/reana-env-root6:6.18.04
+==> Command: mkdir -p results && root -b -q 'code/gendata.C(20000,"results/data.root")'
+==> Status: finished
+==> Started: 2021-11-26T12:46:17
+==> Finished: 2021-11-26T12:46:23
+==> Logs:
+...
+```
+
+If you would like to see a brief overview of all the steps of the
+workflow in one go, you can simply grep the '==>' string in the `logs`
+command output:
+
+```console
+$ reana-client logs -w my-analysis.42 | grep '^==>'
+==> Workflow engine logs
+==> Job logs
+==> Step: gendata
+==> Workflow ID: 4455a6b2-3d94-4694-ae99-e493327cd53f
+==> Compute backend: Kubernetes
+==> Job ID: reana-run-job-6f57f1c8-8edf-4840-9423-b638e664bf57
+==> Docker image: reanahub/reana-env-root6:6.18.04
+==> Command: mkdir -p results && root -b -q 'code/gendata.C(20000,"results/data.root")'
+==> Status: finished
+==> Started: 2021-11-26T12:46:17
+==> Finished: 2021-11-26T12:46:23
+==> Logs:
+==> Step: fitdata
+==> Workflow ID: 4455a6b2-3d94-4694-ae99-e493327cd53f
+==> Compute backend: Kubernetes
+==> Job ID: reana-run-job-2da81be7-5108-41cd-8b7c-ffc0f081b3bd
+==> Docker image: reanahub/reana-env-root6:6.18.04
+==> Command: root -b -q 'code/fitdata.C("results/data.root","results/plot.png")'
+==> Status: finished
+==> Started: 2021-11-26T12:46:23
+==> Finished: 2021-11-26T12:46:33
+==> Logs:
+```
+
+### Removal of support for Python 2.7
+
+Following the discontinuation of Python 2.7 official support, REANA
+0.8 release series stopped supporting Python 2.7 version. Please
+upgrade your `reana-client` installation to use at least Python 3.6.
+
+## What's new for the administrators?
+
+### CPU and Disk quota accounting
+
+REANA 0.8 release series allows cluster administrators to set certain
+CPU and Disk quota usage limits for individual users. This can be set
+by Helm configuration option `quota.enabled`.
+
+Please note that if you do enable CPU and Disk quota monitoring, you
+may want to further configure `quota.termination_update_policy` to
+decide whether the CPU and Disk quota usage consumption for users will
+be calculated immediately after workflow terminates or via a periodic
+cron jobs.
+
+### Improved cluster performance
+
+The cluster performance when running numerous concurrent workflows was
+considerably improved. We have tested running up to thousand of
+concurrent workflows running on clusters of up to three hundred
+nodes. There are new Helm configuration options that allow to
+fine-tune REANA cluster components in these conditions, such as:
+
+- ``reana_server.environment.REANA_WORKFLOW_SCHEDULING_POLICY`` allows
+  to set workflow scheduling policy. The value "fifo", meaning
```

Comment:
allow to set -> "allow setting" or "allow us/you to set"

Reply:
Missing.

Reply:
Added.